src/ohri-form.component.tsx

```diff
@@ -202,14 +202,28 @@ const OHRIForm: React.FC<OHRIFormProps> = ({
             await Promise.all(
               postSubmissionHandlers.map(async ({ postAction, config, actionId }) => {
                 try {
-                  await postAction.applyAction(
-                    {
-                      patient,
-                      sessionMode,
-                      encounters: results.map((encounterResult) => encounterResult.data),
-                    },
-                    config,
-                  );
+                  const encounterData = [];
+                  if (results) {
+                    results.forEach((result) => {
+                      if (result?.data) {
+                        encounterData.push(result.data);
+                      }
+                    });
+                    if (encounterData.length) {
+                      await postAction.applyAction(
+                        {
+                          patient,
+                          sessionMode,
+                          encounters: results.map((encounterResult) => encounterResult.data),
+                        },
+                        config,
+                      );
+                    } else {
+                      throw new Error('No encounter data to process post submission action');
+                    }
+                  } else {
+                    throw new Error('No handlers available to process post submission action');
```

Comment:
If the post submission action has a name / description would be nice to include it as part of the error message.

Reply:
Yah, that is defined in the toast in the catch.
Since the error is thrown, once it's caught, it's wrapped in a toast with the appropriate title.(See screenshot attached)